x-pack/plugins/cloud_security_posture/public/common/navigation/types.ts

```diff
@@ -4,10 +4,13 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
+import type { RouteProps } from 'react-router-dom';
+
 export interface CspNavigationItem {
   readonly name: string;
   readonly path: string;
   readonly disabled?: boolean;
+  readonly exact?: RouteProps['exact'];
```

Comment:
i added this for now so the routing to rules page works. 

this my be tinkered with again at:
- https://github.com/elastic/security-team/issues/2597

the desired outcome IMO is:
 - benchmark is its own route - `/benchmark`
 - rules is a child route of benchmark - `/benchmark/:packageId/:policyId/rules`

so when we visit the rules page, the benchmark is still active on the side navigation and the breadcrumbs show the rules as a benchmark child.